deployment/common/changeset/deploy_link_token.go

```diff
@@ -110,10 +114,25 @@ func deployLinkTokenContractEVM(
 ) (*deployment.ContractDeploy[*link_token.LinkToken], error) {
 	linkToken, err := deployment.DeployContract[*link_token.LinkToken](lggr, chain, ab,
 		func(chain deployment.Chain) deployment.ContractDeploy[*link_token.LinkToken] {
-			linkTokenAddr, tx, linkToken, err2 := link_token.DeployLinkToken(
-				chain.DeployerKey,
-				chain.Client,
+			var (
+				linkTokenAddr common.Address
+				tx            *eth_types.Transaction
+				linkToken     *link_token.LinkToken
+				err2          error
 			)
+			if !chain.IsZk {
+				linkTokenAddr, tx, linkToken, err2 = link_token.DeployLinkToken(
+					chain.DeployerKey,
+					chain.Client,
+				)
+			} else {
+				linkTokenAddr, _, linkToken, err2 = link_token.DeployLinkTokenZk(
```

Comment:
Isn't this zksync specific bytecode? Or do all ZK EVM chains use the same bytecode as ZKSync?

Reply:
There are other chains using this VM to but not necessarily all ZK chains